src/org/wavescale/sourcesync/ui/ConnectionConfigPanel.java

```diff
@@ -107,7 +107,11 @@ public void setConnectionType(String connectionType) {
      * @return an int value representing the target port.
      */
     public int getPort() {
-        return Integer.valueOf(tfPort.getText());
+        try {
+            return Integer.valueOf(tfPort.getText());
+        } catch (NumberFormatException e) {
+            return 22;
```

Comment:
Yeah, those would definitely be better. However, I'm not experienced enough with IntelliJ plugin development to write the improved fix myself.